src/components/typeahead/typeahead.component.ts

```diff
@@ -144,7 +149,11 @@ export class TypeaheadComponent<T = any> implements OnChanges, OnDestroy {
     constructor(
         public typeaheadElement: ElementRef,
         private _changeDetector: ChangeDetectorRef,
-        private _service: TypeaheadService
+        private _popoverOrientation: PopoverOrientationService,

```

Comment:
With the change to store the object, this no longer needs to be persisted, so you can remove the `private` and the underscore.